### PR TITLE
tests: benchmarks: Check idle current with uart30 for MLT

### PR DESCRIPTION
**Pull request by nordic-bami**
Use UART30 for nrf54l15 and verify the idle current with it

### CODE REVIEW

**Review by nordic-pikr — APPROVED** [review]

File: tests/benchmarks/current_consumption/uart_idle/testcase.yaml [hunk 1]
```diff
@@ -14,3 +14,16 @@ tests:
...
+      - nrf54l15dk/nrf54l15/cpuapp
+    integration_platforms:
+      - nrf54l15dk/nrf54l15/cpuapp
+    extra_args:
+      - DTC_OVERLAY_FILE="boards/nrf54l15dk_nrf54l15_cpuapp_uart30.overlay"
+    harness: pytest
+    harness_config:
+      fixture: ppk_power_measure
+      pytest_root:
+        - "${CUSTOM_ROOT_TEST_DIR}/test_measure_power_consumption.py::test_measure_and_data_dump_power_consumption_with_rx_wakeup_54L_uart_idle"
```
Comment (nordic-piks) [inline tests/benchmarks/current_consumption/uart_idle/testcase.yaml:28]: That needs to be different function as idle current is different.

Comment (nordic-bami) [reply tests/benchmarks/current_consumption/uart_idle/testcase.yaml:28]: Not much really, it still fist within the specified limit.

Comment (nordic-piks) [reply tests/benchmarks/current_consumption/uart_idle/testcase.yaml:28]: In this test we want to track this lower value.

File: tests/benchmarks/current_consumption/uart_idle/boards/nrf54l15dk_nrf54l15_cpuapp_uart30.overlay [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ */
+
+/ {
+	chosen {
+		zephyr,shell-uart = &uart30;
+		zephyr,console = &uart30;
+		zephyr,uart-mcumgr = &uart30;
+		zephyr,bt-mon-uart = &uart30;
+		zephyr,bt-c2h-uart = &uart30;
+
```
Comment (nordic-segl) [inline tests/benchmarks/current_consumption/uart_idle/boards/nrf54l15dk_nrf54l15_cpuapp_uart30.overlay:14]: Empty line not needed?

**Review by nordic-segl — APPROVED** [review]

**Pull request merged**